es6-shim.js

```diff
@@ -1096,7 +1098,9 @@
 
   var ArrayPrototypeShims = {
     copyWithin: function copyWithin(target, start) {
-      var end = arguments[2]; // copyWithin.length must be 2
+      // copyWithin.length must be 2, so we can't add `end` to the arguments
```

Comment:
this comment isn't necessary, we should definitely be consistently checking `arguments.length` before accessing an index - which we already do in many places.